content/_index.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
We should either replace the `README.md` of the repository with a short description that this repository contains a website, a link and some instructions on how to build it locally; or symlink it to this location so we aren't copying and pasting the homepage.